runtime/cere-dev/src/lib.rs

```diff
@@ -501,7 +501,6 @@ impl_opaque_keys! {
 		pub babe: Babe,
 		pub im_online: ImOnline,
 		pub authority_discovery: AuthorityDiscovery,
-		pub ddc_verification: DdcVerification,
```

Comment:
Why we are deleting it? Was it the reason of SessionKeys mess ?

Reply:
@yahortsaryk I have corrected it. `ddc_verification` should not be part of `OldSessionKeys`. It was added by mistake in @aie0 's last PR https://github.com/Cerebellum-Network/blockchain-node/pull/402.  

That was not the reason for SessionKeys mess because we haven't migrated session keys in previous PRs.